### DISCUSSION TITLE
Preview image for posts

### DISCUSSION THREAD

**Discussion by dotzenith** [hugo-sid/hugo-blog-awesome#137] [Feature requests]
Hi Sid! Thank your for this awesome theme, it's incredibly elegant and I love almost everything about it. 

I'm opening this discussion to ask if it would be possible to embed preview images to the blog posts. Please let me know if this is already possible and I just missed it. Thank you!

**Comment by hugo-sid** [comment]
Thanks for your kind words @dotzenith.

> I'm opening this discussion to ask if it would be possible to embed preview images to the blog posts.

This feature is not available in the theme currently.
